week-3/refactor/is-vowel.js

```diff
@@ -1,8 +1,15 @@
 function isVowel(letter) {
+  // Check if is a string and change the log text
+  if (typeof letter == "string") {
+    console.log(`Case: letter ${letter}...`);
+  } else {
+    console.log(`Case: ${letter}...`);
+  }
+
   return (
     letter === "a" ||
     letter === "e" ||
-    letter === "i" ||
+    // letter === "i" || repeated comparison
     letter === "i" ||
     letter === "o" ||
     letter === "u"
```

Comment:
I think this whole part of the function could be replaced with an [Array.prototype.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes): 

```
    return ["a", "e", "i", "o", "u"].includes(letter)
```

Reply:
Thank you a lot. This approach is much better, cleaner, and easier to understand.